src/assignment/components/AssignmentTitle.tsx

```diff
@@ -20,18 +20,19 @@ type AssignmentTitleProps = {
 
 const AssignmentTitle: FC<AssignmentTitleProps> = ({ control, setAssignment }) => {
 	const { tText } = useTranslation();
+	const [isActive, setIsActive] = useState<boolean>(false);
 
 	return useMemo(
 		() => (
-			<Flex center className="u-spacer-top-l">
-				<Icon name={IconName.clipboard} size="large" />
+			<Flex center className="u-spacer-top-l c-inline-title-edit">
+				<Icon name={IconName.clipboard} size="large" subtle />
 
-				<BlockHeading className="u-spacer-left" type="h2">
+				<BlockHeading type="h2">
```

Comment:
Why don't we use the CollectionOrBundleTitle component here?
We could call it something like: InlineTitleEditField instead of specifically for collections/bundels